web-app/package.json

```diff
@@ -49,7 +50,8 @@
     "lint": "eslint 'src/app/**/*.{js,ts,tsx}'",
     "lint:fix": "eslint 'src/app/**/*.{js,ts,tsx}' --fix",
     "cypress:run": "cypress run",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "generate:api-types": "npx openapi-typescript ../docs/DatabaseCatalogAPI.yaml -o src/app/services/mobility-database-api.ts"
```

Comment:
I see this script generates files to  `web-app/src/app/services/mobility-database-api.ts` but the types are located in `web-app/src/app/services/feeds/types.ts`, do we want to output it directly to types.ts?

Reply:
Good catch, should be `src/app/services/feeds/types.ts` in the script -- fixed